contrib/autopilot/tasks.py

```diff
@@ -517,7 +533,72 @@
+                            ("news_commentary_v13/training-parallel-nc-v13/news-commentary-v13.de-en.de",
+                             "news_commentary_v13/training-parallel-nc-v13/news-commentary-v13.de-en.en",
+                             TEXT_UTF8_RAW),
+                            ("rapid_eu_2016/rapid2016.de-en.de",
+                             "rapid_eu_2016/rapid2016.de-en.en",
+                             TEXT_UTF8_RAW),
+                        ],
+                        dev=[
+                            ("wmt18_dev/dev/newstest2017-deen-src.de.sgm",
+                             "wmt18_dev/dev/newstest2017-deen-ref.en.sgm",
+                             TEXT_UTF8_RAW_SGML),
+                        ],
+                        test=[
+                            ("wmt18_test/test/newstest2018-deen-src.de.sgm",
+                             "wmt18_test/test/newstest2018-deen-ref.en.sgm",
+                             TEXT_UTF8_RAW_SGML),
+                        ]),
+    "wmt18_en_de": Task(description="WMT18 English--German",
+                        url="http://statmt.org/wmt18/translation-task.html",
+                        src_lang="en",
+                        trg_lang="de",
+                        bpe_op=32000,
+                        train=[
+                            ("europarl_v7/training/europarl-v7.de-en.en",
+                             "europarl_v7/training/europarl-v7.de-en.de",
+                             TEXT_UTF8_RAW),
+                            ("paracrawl_release1_en_de/paracrawl-release1.en-de.zipporah0-dedup-clean.en",
+                             "paracrawl_release1_en_de/paracrawl-release1.en-de.zipporah0-dedup-clean.de",
+                             TEXT_UTF8_RAW),
+                            ("common_crawl_wmt13/commoncrawl.de-en.en",
+                             "common_crawl_wmt13/commoncrawl.de-en.de",
+                             TEXT_UTF8_RAW),
+                            ("news_commentary_v13/training-parallel-nc-v13/news-commentary-v13.de-en.en",
+                             "news_commentary_v13/training-parallel-nc-v13/news-commentary-v13.de-en.de",
+                             TEXT_UTF8_RAW),
+                            ("rapid_eu_2016/rapid2016.de-en.en",
+                             "rapid_eu_2016/rapid2016.de-en.de",
+                             TEXT_UTF8_RAW),
+                        ],
+                        dev=[
+                            ("wmt18_dev/dev/newstest2017-ende-src.en.sgm",
+                             "wmt18_dev/dev/newstest2017-ende-ref.de.sgm",
+                             TEXT_UTF8_RAW_SGML),
+                        ],
+                        test=[
+                            ("wmt18_test/test/newstest2018-ende-src.en.sgm",
+                             "wmt18_test/test/newstest2018-ende-ref.de.sgm",
+                             TEXT_UTF8_RAW_SGML),
+                        ]),
+    # NMT18 workshop shared task
```

Comment:
"WNMT" is the official abbreviation.  If we want to avoid confusion with "WMT", we could expand this to "2018 Workshop on Neural Machine Translation and Generation".